gym/wrappers/env_checker.py

```diff
@@ -0,0 +1,57 @@
+"""A passive environment checker wrapper for an environment's observation and action space along with the reset, step and render functions."""
+from typing import Tuple, Union
+
+import gym
+from gym.core import ActType, ObsType
+from gym.utils.passive_env_checker import (
+    check_action_space,
+    check_observation_space,
+    passive_env_render_check,
+    passive_env_reset_check,
+    passive_env_step_check,
+)
+
+
+class PassiveEnvChecker(gym.Wrapper):
```

Comment:
Why define the Wrapper here and the method implementation in another file?

Reply:
Good question, I think that is because the passive environment checker file has more functions than are used in the wrapper and I use the passive env reset and step check in the full invasive environment checker. So these functions would have to be in a separate place